### PR TITLE
add runFcitx5 button

### PR DESCRIPTION
**Pull request by chenshijie-uos**
add runFcitx5 button

### CODE REVIEW

File: src/lib/configwidgetslib/erroroverlay.cpp [hunk 1]
```diff
@@ -41,6 +44,11 @@ void ErrorOverlay::availabilityChanged(bool avail) {
     }
 }
 
+void ErrorOverlay::runFcitx5()
+{
+    system("fcitx5-autostart");
```
Comment (wengxt) [inline src/lib/configwidgetslib/erroroverlay.cpp:49]: 1. Just run fcitx5.
2. don't use system().
See how FcitxModule::runFcitx() implement it.

File: src/lib/configwidgetslib/erroroverlay.cpp [hunk 2]
```diff
@@ -25,6 +26,8 @@ ErrorOverlay::ErrorOverlay(DBusProvider *dbus, QWidget *parent)
     connect(baseWidget_, &QObject::destroyed, this, &QObject::deleteLater);
     connect(dbus, &DBusProvider::availabilityChanged, this,
             &ErrorOverlay::availabilityChanged);
+
+    connect(ui_->pushButton,&QAbstractButton::pressed,this,&ErrorOverlay::runFcitx5);
```
Comment (wengxt) [inline src/lib/configwidgetslib/erroroverlay.cpp:30]: Code is not properly formatted, please run clang-format.

File: src/lib/configwidgetslib/erroroverlay.ui [hunk 3]
```diff
@@ -211,6 +211,33 @@
...
    </item>
+   <item>
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string/>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QPushButton" name="pushButton">
```
Comment (wengxt) [inline src/lib/configwidgetslib/erroroverlay.ui:222]: maybe we can have a more descriptive name for this

**Review by wengxt — CHANGES_REQUESTED** [review]

**Review by chenshijie-uos — COMMENTED** [review]
i close it ,when i changed right , i pr this ？

**Pull request closed without merging**